src/global.scss

```diff
@@ -8,3 +8,8 @@ html, body {
   height: 100vh;
   width: 100vw;
 }
+
+.page.swiper-slide {
+  display: grid;
+  place-content: center;
+}
```

Comment:
Nope please use bootstrap classes to align center elements.

Reply:
Nope we don't want to write it on each page but use something global.

Reply:
✅ in c791201

Moved to the Page component